RetrieveService/src/main.py

```diff
@@ -29,8 +29,13 @@ async def get_templates(s3=Depends(get_client_s3)) -> JSONResponse:
             Json response with the status code and data containing
             the message and data.
     """
+    prefix = ''
+    if file_type in ('filled', 'template'):
```

Comment:
If the file type is not supported then it should return a response with an error.

Reply:
done